pillar/salt_master.sls

```diff
@@ -179,7 +180,11 @@ salt_master:
         key: use-instance-role-credentials
       engines:
         - sqs_events:
-            queue: {{ sqs_mitxpro_production_queue }}
+            queue: {{ sqs_edxapp_mitxpro_production_queue }}
+            profile: sqs
+            tag: salt/engine/sqs/mitxpro-production-autoscaling
```

Comment:
Should there be different tags for each queue to avoid crossing the threads on which instance type is being managed?

Reply:
Not really sure what we'd gain by it, so left it as is.